sass/custom/_getting_started.scss

```diff
@@ -3,6 +3,10 @@
         padding-bottom: 16px;
     }
 
+    em {
+        font-weight: bold;
```

Comment:
Why not just use `<b>` in that case?

Reply:
Didn't made the font bold, not sure why.

But isn't emphasizing preferred over direct format tags such as `b` these days?

Reply:
dunno, feels weird to introduce another property, but that is me :)

I'm not the CSS expert here, this is more @ludeeus domain.

Reply:
I think `<b>` should be used here, well not here but in the intro text/list.
I don't see a reason it should not work, it works when I do it.